bin/contentful-migration.js

```diff
@@ -59,6 +57,15 @@ const renderFailedValidation = function (errors, renderer) {
 };
 
 const run = Bluebird.coroutine(function * () {
+  let migrationFunction;
+  try {
+    migrationFunction = require(argv.filePath);
+  } catch (e) {
+    console.log(`The ${argv.filePath} script could not be parsed, as it seems to contain syntax errors.\n`);
+    console.log(e);
```

Comment:
What kind of error do we get here? A detailed line number thingy, or only "unexpected token"?

Reply:
the whole error including the trace
![image](https://user-images.githubusercontent.com/9396227/30912323-60d304b8-a38c-11e7-845c-ea49f1da1223.png)

Reply:
Let's make our custom output red and/or bold then, maybe?
For the inattentive dev this looks a lot like something is broken with our tooling.